cart.js

```diff
@@ -1,8 +1,20 @@
 const navCart = document.getElementById("nav__cart");
 const cartMenu = document.getElementById("nav__cart-menu");
+// const navHamburger = document.getElementById("nav__hamburger");
+// const hamburgerMenu = document.getElementById("nav__hamburger-menu");
+// const hamburgerIcon = document.getElementById("nav__hamburger-hamburgerIcon");
+// const closeIcon = document.getElementById("nav__hamburger-closeIcon");
 
 navCart.addEventListener("click", toggleCart);
 
 function toggleCart() {
-  cartMenu.classList.toggle("nav__cart-menu--on");
+  if(hamburgerMenu.classList.contains("nav__hamburger-menu--on")) {
```

Comment:
1) Tabulation and spacing:
if (hamburgerMenu.classList.contains("nav__hamburger-menu--on")) {
    hamburgerMenu.classList.remove("nav__hamburger-menu--on");
    hamburgerMenu.classList.add("nav__hamburger-menu--off");
    closeIcon.classList.remove("nav__hamburger-menu--on");
    closeIcon.classList.add("nav__hamburger-menu--off");
    hamburgerIcon.classList.remove("nav__hamburger-menu--off");
    hamburgerIcon.classList.add("nav__hamburger-menu--on");
}

Do you know that property of css can be overwritten by another class?
It will reduce amount of twice